Library/Homebrew/test/.rubocop.yml

```diff
@@ -2,4 +2,3 @@ inherit_from:
   - ../.rubocop.yml
   - .rubocop_todo.yml
 
-require: rubocop-rspec
```

Comment:
Can the `RSpec` section above be in this file?

Reply:
It looks like it, pushing now.